qa/sc_import_export_keys.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from test_framework.util import assert_equal, assert_false, assert_true, initialize_chain_clean, start_nodes, websocket_port_by_mc_node_index, forward_transfer_to_sidechain
+from SidechainTestFramework.scutil import start_sc_nodes, generate_next_blocks, bootstrap_sidechain_nodes, connect_sc_nodes
+from httpCalls.wallet.createPrivateKey25519 import http_wallet_createPrivateKey25519
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from httpCalls.wallet.exportSecret import http_wallet_exportSecret
+from httpCalls.wallet.allPublicKeys import http_wallet_allPublicKeys
+from httpCalls.wallet.importSecret import http_wallet_importSecret
+from httpCalls.wallet.dumpSecrets import http_wallet_dumpSecrets
+from httpCalls.wallet.importSecrets import http_wallet_importSecrets
+from httpCalls.wallet.balance import http_wallet_balance
+from httpCalls.transaction.sendCoinsToAddress import sendCoinsToAddress
+from SidechainTestFramework.sidechainauthproxy import SCAPIException
+
+"""
+    - Setup 2 SC Node.
+    - Create a new address on node2
+    - Export its private key and import it inside the node1
+    - Verify that node1 owns this address
+    - Send coins to this address and verify that the balance on node1 is unchanged and node2 owns the new amount
```

Comment:
I guess here: "Send coins from node1 to this address...."
That is the reason why the balance of node1 is unchanged. Correct?

Reply:
Yes correct, I updated the comment.